docs/docs/examples/guides/object_detection_example.md

```diff
@@ -0,0 +1,610 @@
+---
+sidebar_position: 6
+title: Using YOLO to Track Objects
+---
+
+import rvizgif from './assets/object-detection-example-rviz.gif'
+
+# Using YOLO to track objects
+
+This example provides a use case for `Object Detection Components`. We show how to create a custom component which
+converts bounding boxes into a 3D pose. The example works with either an Intel Realsense camera using
+`collections/intel-realsense-collection` or the Camera Streamer from `components/core-vision`. The final application will be able to track an object with a robot based on a fixed camera position.
+
+<div class="text--center">
+  <img src={rvizgif} alt="Moving the robot towards an object in RVis" />
+</div>
+
+
+## Setup
+To run a YOLO executor, you need a yolo model file in ONNX format and a YAML class file. Yolo models are widely available in PT formats. For example, download the lightweight YOLO12n.pt from ultralytics [here](https://github.com/sunsmarterjie/yolov12).
+
+To convert a PT file to ONNX, run the following Python code with ultralytics installed:
+
+```python
+from ultralytics import YOLO
+model = YOLO("yolov12n.pt")
+model.export(format="onnx")  # creates 'yolov12n.onnx'
+```
+
+For this example, also download the standard coco.yaml class file [here](https://github.com/ultralytics/ultralytics/blob/main/ultralytics/cfg/datasets/coco.yaml).
+Move the ONNX model file and the YAML class file into a new directory.
+
+In AICA Launcher, create a configuration with the following core version and packages:
+
+- AICA Core v4.3.2
+- `collections/object-detection` (TODO: specify version)
+- `collections/intel-realsense-collection v1.0.0`
+- `components/core-vision v1.0.0`
+
+Under Advanced Settings, add a volume containing the folder where you stored your YOLO model files and link it to `/files`.
+
+## Using the YOLO Executor
+
+The *YOLO Executor* component observes runs the YOLO segmentation model on an image. 
+It takes a camera stream and outputs the segmented image as well as the locations of bounding boxes on the image. The user can set the `Object Class` parameter, which will cause the component to set the predicate `Object Detected` to `True` when the specified object is found in the image. The `Confidence Threshold` parameter is the minimum score a predicted bounding box must have to be considered a valid detection. `IOU Threshold` is used during Non-Maximum Suppression (NMS) to decide whether two bounding boxes represent the same object. For example, if `IOU threshold` is set to 0.5, any box that overlaps more than 50% with a higher-scoring box will be discarded. For now we leave them as default values.
+
+To test the YOLO executor:
+- Remove the `*Hardware Interface*`.
+- Add the following components:
+  - `Realsense2 Camera/Realsense Camera` if you have an Intel Realsense camera.
```

Comment:
I think we can Just add the component name (not the category name) to remove a bit of visual clutter, but I'm not sure everyone will agree. For example, we could perhaps mention the components and their category title in the setup, and from that point forward just mention them by component name only